src/App.tsx

```diff
@@ -1,26 +1,42 @@
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import { Loader, TodoFilter, TodoList, TodoModal } from './components';
+import { useAppDispatch, useAppSelector } from './app/store';
+import { useEffect, useState } from 'react';
+import { getTodos } from './api';
+import { todosSlice } from './features/todos';
+import { isSelected } from './features/currentTodo';
 
-export const App = () => (
-  <>
-    <div className="section">
-      <div className="container">
-        <div className="box">
-          <h1 className="title">Todos:</h1>
+export const App = () => {
+  const todoModal = useAppSelector(isSelected);
+  const dispatch = useAppDispatch();
 
-          <div className="block">
-            <TodoFilter />
-          </div>
+  const [isLoading, setIsLoading] = useState(false);
+
+  useEffect(() => {
+    setIsLoading(true);
+    getTodos()
+      .then(resp => dispatch(todosSlice.actions.setTodos(resp)))
+      .catch()
```

Comment:
The `catch()` block is empty. It's important to handle errors properly to avoid silent failures. Consider logging the error or displaying an error message to the user.